audio_recorder/src/main.py

```diff
@@ -14,10 +14,10 @@
 
     recorder = Recorder()
     audio_properties = Audio_properties()
-    output_file_path = "../output/test.wav"
+    output_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "output", "test.wav")
 
     udp_sender = UDPSender()
-    udp_sender.starting_server_message()
```

Comment:
I don't like that we do not have start/stop messages any more. In a case where audio_recodred fail for some reason we can have invalid status by just toggling presence. This error condition is not checked elsewhere so we can have follow up problems. Please fix.